src/main/scala/uk/gov/nationalarchives/XMLCreator.scala

```diff
@@ -3,9 +3,10 @@ package uk.gov.nationalarchives
 import cats.effect.IO
 import uk.gov.nationalarchives.DynamoFormatters._
 
+import java.time.OffsetDateTime
 import scala.xml._
 
-class XMLCreator {
+class XMLCreator(ingestDateTime: () => OffsetDateTime) {
```

Comment:
Does this need to be a function? Can you not just pass in the `OffsetDateTime`? You can then pass in a static `OffsetDateTime` in the tests?

Reply:
Ah, yes, you're right; I passed it as a function in order to get a later time but it's not as though a few milliseconds/microseconds is going to matter